pkgs/applications/networking/instant-messengers/weechat-matrix-bridge/library-path.patch

```diff
@@ -0,0 +1,13 @@
+diff --git a/matrix.lua b/matrix.lua
+index b79f500..c731ce4 100644
+--- a/matrix.lua
++++ b/matrix.lua
+@@ -43,6 +43,8 @@ This script maps this as follows:
+ 
+ ]]
+ 
++package.cpath = package.cpath .. ";__NIX_LIB_PATH__"
++
+ local json = require 'cjson' -- apt-get install lua-cjson
+ local olmstatus, olm = pcall(require, 'olm') -- LuaJIT olm FFI binding ln -s ~/olm/olm.lua /usr/local/share/lua/5.1
```

Comment:
Does not this also require `libolm`?

Reply:
good point, my current (testing) setup works without olm ATM.
Will update the expression as soon as I got time to :-)

Reply:
fixed